docs/modules/operation/pages/admin/webui/grafana-dashboard-box.adoc

```diff
@@ -72,10 +72,30 @@ When no tag is given all dashboards will be displayed
 | Maximum number of entries to be displayed (0 for unlimited)
 | Integer
 | 0
+
+| org.opennms.grafanaBox.link.protocol
+| The protocol used to build each dashboard link URL.
+| String
+| org.opennms.grafanaBox.protocol
+
+| org.opennms.grafanaBox.link.hostname
+| Hostname used to build each dashboard link URL.
+| String
+| org.opennms.grafanaBox.hostname
+
+| org.opennms.grafanaBox.link.port
+| The port used to build each dashboard link URL.
+| Integer
+| org.opennms.grafanaBox.port
+
+| org.opennms.grafanaBox.link.basePath
+| The Grafana base path used to build each dashboard link URL.
+| String
+| org.opennms.grafanaBox.basePath
 |===
 
-TIP: If you have Grafana behind a proxy it is important the `org.opennms.grafanaBox.hostname` is reachable.
-     This host name is used to generate links to the Grafana dashboards.
+TIP: If you have Grafana behind a reverse proxy, you may need to specify a different set of values for the sets of properties with link in their names versus the ones without link.
+The properties with link in their names, if set, are used to build the links that are shown to users; the other properties are used to construct the URLs used for internal interaction with the Grafana REST API.
 
 The process to generate a Grafana API Key can be found in the https://grafana.com/docs/grafana/latest/http_api/[HTTP API documentation].
```

Comment:
I would suggest moving this line between lines 46 and 47 so that it can be part of the cell talking about the `aiKey` property instead of as a footnote.  Also, if you wouldn't mind making the descriptions for all the property descriptions in the table full sentences (add periods at the end), that would be appreciated.
```suggestion
```